pubspec.yaml

```diff
@@ -4,26 +4,26 @@ description: Easy and Fast internationalizing and localization your Flutter Apps
 homepage: https://github.com/aissat/easy_localization
 issue_tracker: https://github.com/aissat/easy_localization/issues
 
-version: 2.3.1-dev.2
+version: 3.0.0-dev.0
 
 environment:
-  sdk: ">=2.6.0 <3.0.0"
+  sdk: ">=2.7.0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  intl: ^0.16.0
-  shared_preferences: ^0.5.7
-  args: ^1.5.2
-  path: ^1.6.4
+  intl: ">=0.16.0 <=0.17.0-nullsafety.2"
```

Comment:
">=0.16.0  <0.17.0" its not null saety version

Reply:
https://github.com/aissat/easy_localization/pull/312#discussion_r553276058

Reply:
@Overman775  check this https://github.com/flutter/flutter/issues/49931

Reply:
#288